README.md

```diff
@@ -22,6 +22,7 @@ Full documentation is available at https://spl.solana.com
 
 1. Install the latest Rust stable from https://rustup.rs/
 2. Install Solana v1.4.7 or later from https://docs.solana.com/cli/install-solana-cli-tools
```

Comment:
Yikes, this is stale.   While you're in here, would you mind bumping this to v1.6.1?

Reply:
done